source/installer.py

```diff
@@ -515,48 +515,116 @@ def isDesktopShortcutInstalled():
 	return os.path.isfile(shortcutPath)
 
 
-def unregisterInstallation(keepDesktopShortcut: bool = False) -> None:
+def _unregisterEaseOfAccessApp():
 	try:
 		winreg.DeleteKeyEx(
 			winreg.HKEY_LOCAL_MACHINE,
 			RegistryKey.EASE_OF_ACCESS_APP.value,
-			winreg.KEY_WOW64_64KEY,
+			# TODO: remove when NVDA is 64-bit only.
+			access=winreg.KEY_WOW64_64KEY,
 		)
+	except WindowsError:
+		log.debug("Ease of Access app key not found nothing to unregister.")
+	try:
 		easeOfAccess.setAutoStart(easeOfAccess.AutoStartContext.ON_LOGON_SCREEN, False)
 	except WindowsError:
-		pass
+		log.debug("Could not disable auto start on logon screen.")
+
+
+def _unregisterDesktopShortcut(keepDesktopShortcut: bool):
```

Comment:
Historically we've tended not to bother with `None` for return types, should I add it to the other functions too?